sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -109,11 +109,17 @@ def test_truncation_slices(eve_test_ts,
 
 def test_truncation_timerange(eve_test_ts):
     truncated = eve_test_ts.truncate(eve_test_ts.time_range.split(3)[1])
+    truncated_1 = eve_test_ts.truncate('2016-06-10 00:02:00', '2016-06-10 00:06:00')    
     # Check the resulting timerange in both TS and TSMD
     assert (truncated.time_range ==
             truncated.meta.time_range ==
             eve_test_ts.time_range.split(3)[1])
-
+    # Check when the timerange is in str
+    assert truncated_1.time_range == truncated_1.meta.time_range == TimeRange('2016-06-10 00:02:00', '2016-06-10 00:06:00')    
+    # Check when the timerange does not overlaps with the data timerange    
+    with pytest.raises(ValueError, match="Provided timerange is not within the bounds of the timeseries"):
+        eve_test_ts.truncate(TimeRange('2012-06-07 05:00', '2012-06-07 06:30'))
```

Comment:
I've make sure that `TimeRange('2012-06-07 05:00', '2012-06-07 06:30')` lies out of `TimeRange`

Do i have to write `TimeRange(eve_test_ts.time_range.start - 2*u.d, eve_test_ts.time_range.start - 1*u.d )` ?

Reply:
If you want? Is it the same test?

Reply:
```
    with pytest.raises(ValueError, match="Provided timerange is not within the bounds of the timeseries"):
        eve_test_ts.truncate(TimeRange('2012-06-07 05:00', '2012-06-07 06:30'))
```
I think we don't need this test since we are also using Same test here

https://github.com/sunpy/sunpy/blob/9b56f853ce9c6dff35d0474d36ae8ec8a298ebcb/sunpy/timeseries/tests/test_timeseriesbase.py#L214-L216